react-redux/contactsMVC/components/commentbox.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import { List, ListItem } from 'material-ui/List';
+import FlatButton from 'material-ui/FlatButton';
+import Avatar from 'material-ui/Avatar'
+
+class CommentBox extends React.Component {
+  render() {
+    const { id, comment, username, thumbnail } = this.props.commentObject;
```

Comment:
you can do this when all variable names are short. But when they are longer, you may
```javascript
const {
  id,
  comment,
  username,
  thumbnail,
} = this.props.commentObject
```